### ISSUE TITLE
Quarks API Reference guide?

### ISSUE THREAD

**Issue by jms90h5** [quarks-edge/quarks.documentation#27]
It's quite likely that I just missed it, but I've looked everywhere I can think of to find the [comprehensive] Quarks reference guide but come up empty.  Basically what I was expecting to find was something similar to the Streams Java API reference guide (like is available in the Streams Knowledge Center for each release).  Or barring that even a description of what how quarks is different from that API (i.e. missing or different).

**Comment by kathyssaunders** [comment]
I can point you to some of the information you may need, that you might have found in an API reference guide.  

Javadoc - http://quarks-edge.github.io/quarks/docs/javadoc/index.html.  A lot of the reference material is here.  

Getting Started - This is not really reference information, but it does walk you through creating a simple program.  

Sample Programs - http://quarks-edge.github.io/quarks.documentation//docs/quarks/samples/ (there is also javadoc for the samples)

This is the very first Quarks release, and more information could be provided to help application developers, javadoc improved, etc.  It would be great if you can review the available information, particularly the javadoc, and then post some more details on what you would like to see in a reference guide or information you think is missing.  It would also be great if you are interested in helping to contribute to the documentation as well!

**Comment by jms90h5** [comment]
Hi Kathy,

Thanks for the links. the javadoc stuff was exactly what I was looking for.  In order to make it easier for others to find I think it would be good to add that link to the Quarks documentation page.  [http://quarks-edge.github.io/quarks.documentation/](url)  I somehow missed it when looking around in the github repository where I downloaded the platform and it isn't directly referenced on the supporting (Documentation) web page which is the second place I looked.
